scripts/kubevirt/repo_groups.sql

```diff
@@ -141,14 +52,11 @@ set
 
 update gha_repos set alias = 'kubevirt/kubevirt' where name like '%kubevirt' or name = 'kubevirt/';
 
-insert into gha_repo_groups(id, name, alias, repo_group, org_id, org_login) select id, name, alias, coalesce(repo_group, name), org_id, org_login from gha_repos on conflict do nothing;
-insert into gha_repo_groups(id, name, alias, repo_group, org_id, org_login) select id, name, alias, org_login, org_id, org_login from gha_repos where org_id is not null and org_login is not null and trim(org_login) != '' on conflict do nothing;
-
 select
   repo_group,
   count(*) as number_of_repos
 from
-  gha_repo_groups
+  gha_repos
```

Comment:
We don't use `gha_repos` for repo groups anymore, so I cannot approve this script. Also it seems to make a LOT of changes - this needs both fixing and approval from K8s community, marking as blocked and requesting changes.

Reply:
@lukaszgryglicki apologies for not noticing this until now. I've created an issue to track this here: https://github.com/kubevirt/community/issues/330

Reply:
No problem! have a great day!